src/components/Pages/index.js

```diff
@@ -319,8 +309,7 @@ const VmEditPageConnected = connect(
     vms: state.vms,
   }),
   (dispatch) => ({
-    getAvailableCDImages: () => dispatch(getIsoStorageDomains()),
-    getVmById: (vmId) => dispatch(selectVmDetail({ vmId })),
+    refreshPage: (vmId) => dispatch(refreshDialogPage({ vmId })),
```

Comment:
I assume there is a problem here because while I get an initial page blink from the "Edit" button on a VM, it errors out after the initial render.